app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -296,6 +299,21 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
                     }
                 }
 
+                override fun onShowFileChooser(
+                    view: WebView?,
+                    uploadMsg: ValueCallback<Array<Uri?>?>,
+                    fileChooserParams: FileChooserParams?
+                ): Boolean {
+                    val i = Intent(Intent.ACTION_GET_CONTENT)
+                    i.addCategory(Intent.CATEGORY_OPENABLE)
+                    i.type = "*/*"
+                    mFilePathCallback = uploadMsg
+
+                    val chooserIntent = Intent.createChooser(i, resources.getString(R.string.select_file))
+                    startActivityForResult(chooserIntent, FILE_CHOOSER_RESULT_CODE)
+                    return true
```

Comment:
```suggestion
                    mFilePathCallback = uploadMsg
                    startActivityForResult(fileChooserParams.createIntent(), FILE_CHOOSER_RESULT_CODE)
                    return true
```

Reply:
So if we do not provide an override for the intent type then the user is unable to select any image locally asides from a screenshot.  They are able to go to other apps like Photos but on device images are blocked.

![image](https://user-images.githubusercontent.com/1634145/94585126-6a4e3f80-0234-11eb-8994-902fb699da3c.png)

I think it is ok to override the type as the front end can handle it if we send an incorrect file type.

![image](https://user-images.githubusercontent.com/1634145/94585191-8356f080-0234-11eb-976c-decbca42582e.png)

If we use the suggested intent without modification the user only sees screenshots and not recent photos

![image](https://user-images.githubusercontent.com/1634145/94585243-98cc1a80-0234-11eb-833a-ed9079c22bfa.png)

Notice the date/time above compared to showing all files

![image](https://user-images.githubusercontent.com/1634145/94585273-a41f4600-0234-11eb-93c2-c4ccab89c536.png)